doc/source/deployment/requirements.rst

```diff
@@ -94,6 +94,20 @@ Network Requirements
     separate network should be present on the Caas Platform and ses_config.yml
     mon_host: section.
 
+* Tunnel Network
+    A network defined by its network interface. It must be up and have ip
+    address defined. This is used for tunneling vxlan traffic for tennant vm's
+    within the cloud.
+
+* External Network
+    A network defined by its network interface. This interface should not have
+    an IP address defined (will not be accessable when cloud deployed). The
+    network interface should be configured BOOTPROTO=none and active.
```

Comment:
accessible
should be configured `BOOTPROTO=none`

Reply:
fixed